### PR TITLE
Extend ReactJS resource component for new API enpoint

### PR DESCRIPTION
**Pull request by richpeters**
Adds ordering to resource list. `/test` to see dynamic resource list at bottom (can be ordered)

### CODE REVIEW

File: apps/api/v1/serializers.py [hunk 1]
```diff
@@ -65,11 +71,11 @@ class Meta:
...
-    full_name = serializers.SerializerMethodField()
+    title = serializers.SerializerMethodField()
 
     class Meta:
         model = User
-        fields = ('id', 'full_name',)
+        fields = ('id', 'title',)
 
-    def get_full_name(self, obj):
+    def get_title(self, obj):
```
Comment (zatan) [inline apps/api/v1/serializers.py]: Why did you rename this to `get_title` it sounds like tries to get `Mr`, `Mrs`? When the Django originally called this method `get_full_name`

Comment (richpeters) [reply apps/api/v1/serializers.py]: So I could call title in react on the three search filters, i.e. `tags.title`, `organisation.title`, `user.title`

Comment (richpeters) [reply apps/api/v1/serializers.py]: Mostly for DRY purposes but could see the issue

File: static/src/js/index.jsx [hunk 2]
```diff
@@ -6,40 +6,74 @@ import { Search } from './search/search';
 import { Resource } from './resource/resource';
 import { ResourceFilter } from './resource/resource_filter';
 
-const api = '/api/v1/resources/?format=json'
+let api = '/api/v1/resources/?format=json'
```
Comment (zatan) [inline static/src/js/index.jsx]: Why did you change to `let`? From the docs I can see that:

`const` is a signal that the variable won’t be reassigned.

`let` is a signal that the variable may be reassigned

Comment (richpeters) [reply static/src/js/index.jsx]: Good catch, was gonna change this in a function but decided against it
